.github/workflows/release.yml

```diff
@@ -91,17 +124,26 @@ jobs:
           # Wait for the release script to finish
           wait $RELEASE_PID
           
+          if [ "$GPG_PRIVATE_KEY" = "not_used" ]; then
+            zip docker-build.zip -r spark-release/docker-build.log
+            zip output.zip -r spark-release/output
+          else:
+            zip -P $ASF_PASSWORD docker-build.zip -r spark-release/docker-build.log
+            zip -P $ASF_PASSWORD output.zip -r spark-release/output
+          fi
+
           # Once release is done, kill the tail processes
           kill $TAIL_PID1 $TAIL_PID2   
```

Comment:
To be safer, we could just disable all the console output at all when it is not a dry run.